code/part-two/README.md

```diff
@@ -166,43 +166,43 @@ working with Javascript, start with these documents:
-    * [The REST API](https://sawtooth.hyperledger.org/docs/core/releases/1.0/architecture/rest_api.html)
+    * [REST API](https://sawtooth.hyperledger.org/docs/core/releases/1.0/architecture/rest_api.html)
 - The Javascript SDK
     * [Building and Submitting Transactions](https://sawtooth.hyperledger.org/docs/core/releases/1.0/_autogen/sdk_submit_tutorial_js.html)
     * [Transaction Processor Tutorial](https://sawtooth.hyperledger.org/docs/core/releases/1.0/_autogen/sdk_TP_tutorial_js.html)
 
 ## The Project
 
-When building an application on Sawtooth, much of the nitty gritty of running
+When building an application on Sawtooth, much of the nitty-gritty of running
 and validating a blockchain is handled for you. You won't have to verify
-signatures or hashes, or validate blocks, or confirm consensus. Instead, you
+signatures and hashes, validate blocks, or confirm consensus. Instead, you
 must consider how you can break up the functionality of your application into
 discrete transaction payloads, and how these payloads will alter state data.
 The typical Sawtooth workflow looks something like this:
 
-1. The user initializes some action (i.e. _"set 'a' to 1"_)
+1. The user initializes some value (i.e., _"set 'a' to 1"_)
 2. The _client_ takes that action and:
     - encodes it in a payload (maybe simply `{"a": 1}`)
     - wraps that payload in a signed transaction and batch
     - submits it to the validator
 3. The validator confirms the transaction and batch are valid
 4. The _transaction processor_ receives the payload and:
     - decodes it
-    - verifies it is a valid action (i.e. 'a' _can_ be set to 1)
+    - verifies it is a valid action (i.e., 'a' _can_ be set to 1)
     - modifies state in a way that satisfies the action
       (perhaps address _...000000a_ becomes `1`)
 5. Later, the client might read that state, and decode it for display
 
-So all you are responsible for building two components (the client and the
-transaction processor), and for keeping those components in agreement on how to
+So, you are responsible only for building two components (the client and the
+transaction processor) and for keeping those components in agreement on how to
 encode payloads and state. This application-wide logic is typically referred to
-in Sawtooth with the general term: "transaction family".
+in Sawtooth with the term "transaction family".
 
 For Cryptomoji, the client and processor are each in their own directory, with
-their own tests, and their own READMEs. There is no particular order they need
-to be built in, but we recommended you develop them in parallel based on
-functionality. For example, implement "collection creation" on _both_ the
-client and the processor before moving on to "sire selection" on _either_.
+their own tests and their own READMEs. There is no particular order for building
+them, but we recommended that you develop them in parallel based on
```

Comment:
"for building them" loses the context of what is usually a strictly ordered set of tests. I think it is important to communicate specifically that there are no order requirements between the client and processor.

Reply:
Hmmm.  My intent was to simplify the sentence structure.  How about clarifying this important info with a rewrite?  I propose: 
  
"You do not have to build the client and processor in any particular order, but we recommend that you develop them in parallel, based on ..."

Reply:
The more I look at this sentence, the more I fundamentally hate it. How about a total rewrite:

"You should develop the components in parallel on a feature-by-feature basis."

Reply:
Yes - this is much better. I'll make the change.